code/development-env/README.md

```diff
@@ -199,7 +184,7 @@ kubectl proxy --address 0.0.0.0 --accept-hosts '.*'
 * Start Mongo, DataLab App, DataLab Api, Infrastructure Api and Auth services.
 
 ```bash
-docker-compose -f ./docker/docker-compose-mongo.yml -f ./docker/docker-compose-app.yml -f ./docker/docker-compose-proxy.yml up -d --remove-orphans
+docker-compose -f ./docker/docker-compose-mongo.yml -f ./docker/docker-compose-app.yml -f ./docker/docker-compose-proxy.yml up --remove-orphans
```

Comment:
I prefer having these run in the background so that they are not tied to a particular terminal window. Might be worth adding a note that they could be run in the background using the `-d` flag?

Reply:
Will do.